packages/types/src/localization.ts

```diff
@@ -282,6 +282,7 @@ type _LocalizationResource = {
       title: LocalizationValue;
       imageFormTitle: LocalizationValue;
       imageFormSubtitle: LocalizationValue;
+      destructiveActionSubtitle: LocalizationValue;
```

Comment:
Do you think we can group the delete button key with the rest, by prefixing it with `imageForm`? eg: 
```suggestion
      imageFormDestructiveAction: LocalizationValue;
```

Reply:
@nikosdouvlis probably `imageFormDestructiveActionSubtitle` is better since it matches the convention